src/creature.cpp

```diff
@@ -2045,6 +2045,40 @@ static void set_part_helper( Creature &c, const bodypart_id &id,
     }
 }
 
+bodypart_id Creature::get_part_id( const bodypart_id &id ) const
+{
+    auto found = body.find( id.id() );
+    if( found == body.end() ) {
+        // try to find an equivalent part in the body map
+        for( const auto &bp : body ) {
```

Comment:
Are those types too complexe to be explicited?

Reply:
I think in general we avoid using auto as much as possible, usually makes the code easier to read